auth_signup_verify_email/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Verify email at signup",
```

Comment:
```suggestion
    "name": "Auth Signup Verify Email",
```

Reply:
Why? The name is correct. It mustn't be the "technical name".

Reply:
Ok, but at least the full name could be put in uppercase.

Reply:
I don't like that uppercasing as well... and it's not mandatory in English (but tolerated).